index.html

```diff
@@ -1024,12 +1040,14 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
 <section>
   <h2>PaymentResponse interface</h2>
       <pre class="idl">
-        enum PaymentComplete { "success", "fail", "" };
+        enum PaymentStatus { "success", "fail", "processing", "" };
```

Comment:
I believe we want to remove the `processing` from here pending a better understanding on how the merchant would get notified/retrieve that the payment is succeeded?